### DISCUSSION TITLE
How to solve location information error?

### DISCUSSION THREAD

**Discussion by Anton-Koh** [community/community#37698] [GitHub Education] [answered]
I’m trying to proof my academic status.
But I always make this problem.
![1F374FC2-DDBD-4CC3-96FD-E32C7F515002](https://user-images.githubusercontent.com/44058105/198818850-00f903cf-e9cb-45ea-86bc-9f218da42078.jpeg)
I tried to allow settings in browser settings, tried it on mobile, and tried it in secret mode, but it didn't solve it.
What can I do instead?

**Comment by Anton-Koh** [comment, marked as answer]
….and it solved

**Reply by samus-aran** [reply]
Hi @MINILOBIN 

That's awesome you were able to solve this. Are you able to explain to others in the community how you were able to confirm your browser location?

**Comment by Mr-Zakir789** [comment]
![Screenshot_2022-10-31-20-05-13-217_com android chrome](https://user-images.githubusercontent.com/110555408/199034168-879d7faa-06c8-484b-a63d-cdbdd384b2a7.jpg)

**Reply by Mr-Zakir789** [reply]
How to solve this sir?

**Reply by Mr-Zakir789** [reply]
Sir how to go through this
